ThePensionsRegulator.Frontend/Styles/_tpr-button.scss

```diff
@@ -18,17 +18,15 @@ $tpr-button-padding: 20px;
     line-height: govuk-px-to-rem(33);
     border-radius: 8px;
     box-shadow: none;
-    background: $govuk-button-background-colour url(/_content/ThePensionsRegulator.Frontend/tpr/chevron_right_white_24dp.svg) right center no-repeat;
```

Comment:
You can delete the file too, unless it's the same one used on the start now button.